pyproject.toml

```diff
@@ -42,30 +42,21 @@ dynamic = ["version"]
 [project.optional-dependencies]
 test = [
   "pytest>=7.0",
-  "gym~=0.23",
+  "gymnasium==1.0.0a1",
 ]
 
 [project.urls]
 homepage = "https://github.com/mgbellemare/Arcade-Learning-Environment"
 documentation = "https://github.com/mgbellemare/Arcade-Learning-Environment/tree/master/docs"
 changelog = "https://github.com/mgbellemare/Arcade-Learning-Environment/blob/master/CHANGELOG.md"
 
-[project.scripts]
-ale-import-roms = "ale_py.scripts.import_roms:main"
-
-[project.entry-points."gym.envs"]
-ALE = "ale_py.gym:register_gym_envs"
-__internal__ = "ale_py.gym:register_legacy_gym_envs"
-
 [tool.setuptools]
 packages = [
   "ale_py",
   "ale_py.roms",
-  "ale_py.env",
-  "ale_py.scripts"
 ]
-package-dir = {ale_py = "src/python", gym = "src/gym"}
-package-data = {"ale_py" = ["py.typed", "*.pyi", "**/*.pyi"], "ale_py.roms" = ["*.bin", "md5.txt"]}
+package-dir = {ale_py = "src/python"}
+package-data = {"ale_py" = ["py.typed", "*.pyi", "**/*.pyi"], "ale_py.roms" = ["md5.json", "Roms.tar.gz.b64"]}
```

Comment:
Why include the tar.gz instead of just the bin files? I don't really like this.

Reply:
@jjshoots You preferred to include the tar and not the bins, do you remember why?

Reply:
@pseudo-rnd-thoughts Someone along the chain of command mentioned we cannot have the bins on the GH repo.